unabridged-works-volume-1/original/06-jew-outward.adoc

```diff
@@ -809,12 +835,17 @@ Consider the main ones a little.
 
 Their great exceptions against our doctrine are:
```

Comment:
Should this be offset? It looks a little funny either way in the PDF...perhaps a touch better with the offset, but idk.

Reply:
Yeah, when I was looking at the PDF it looked a little sad.  I think let's have you change it to an offset.